migration/planning-migration-3-to-4.adoc

```diff
@@ -0,0 +1,168 @@
+
+If you want to add RHEL workers to your {product-title} 4.2 cluster, you use an Ansible playbook to join the RHEL workers after the cluster is running. For more information, see xref:../machine_management/adding-rhel-compute.adoc#adding-rhel-compute[Adding RHEL compute machines to an OpenShift Container Platform cluster].
+
+[discrete]
+==== Infrastructure options
+
+In {product-title} 3.11, you installed your cluster on infrastructure that you prepared and maintained. In addition to providing your own infrastructure, {product-title} 4 offers an option to deploy a cluster on infrastructure that the {product-title} installation program provisions and the cluster maintains.
+
+For more information, see xref:../architecture/architecture-installation.adoc#installation-overview_architecture-installation[OpenShift Container Platform installation overview].
+
+[discrete]
+==== Upgrading your cluster
+
+In {product-title} 3.11, you upgraded your cluster by running Ansible playbooks. In {product-title} 4.2, The cluster manages its own updates, including updates to Red Hat Enterprise Linux CoreOS (RHCOS) on cluster nodes. You can easily upgrade your cluster by using the web console or by using the `oc adm upgrade` command from the OpenShift CLI and the Operators will automatically upgrade themselves. If your {product-title} 4.2 cluster has Red Hat Enterprise Linux workers, then you will still need to run an Ansible playbook to upgrade those workers.
+
+For more information, see xref:../updating/updating-cluster-between-minor.adoc#updating-cluster-between-minor[Updating clusters].
+
+[id="migration-considerations"]
+== Migration considerations
+
+Review the changes and other considerations that might affect your migration from {product-title} 3.11 to {product-title} 4.
+
+[id="migration-preparing-storage"]
+=== Storage considerations
+
+Review the storage changes to consider when migrating from {product-title} 3.11 to {product-title} 4.2.
+
+[discrete]
+==== TODO
+
+[id="migration-preparing-networking"]
+=== Networking considerations
+
+Review the networking changes to consider when migrating from {product-title} 3.11 to {product-title} 4.2.
+
+[discrete]
+==== Network isolation mode
+
+The default network isolation mode for {product-title} 3.11 was `ovs-subnet`. The default network isolation mode for {product-title} 4.2 is now NetworkPolicy. If your {product-title} 3.11 cluster used the `ovs-subnet` or `ovs-multitenant` mode, it is recommended to switch to the NetworkPolicy mode for your {product-title} 4.2 cluster. If you want to maintain the `ovs-multitenant` behavior while using NetworkPolicy in {product-title} 4.2, follow the steps to xref:../networking/configuring-networkpolicy.adoc#nw-networkpolicy-multitenant-isolation_configuring-networkpolicy-plugin[configure multitenant isolation using NetworkPolicy].
```

Comment:
For all practical purposes, the default was "multitenant" in 3.x.

It's probably worth adding that NetworkPolicy is more flexible, supported upstream, and can do everything Multitenant does.